tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/isolation/IsolatedObjectTest.java

```diff
@@ -255,22 +248,15 @@ public void testIsolatedObjectIsolationNegative() {
         validateError(result, i++, ERROR_INVALID_TRANSFER_OUT_IN_LOCK_WITH_RESTRICTED_VAR_USAGE, 748, 31);
         validateError(result, i++, ERROR_INVALID_TRANSFER_OUT_IN_LOCK_WITH_RESTRICTED_VAR_USAGE, 748, 57);
         validateError(result, i++, ERROR_INVALID_TRANSFER_OUT_IN_LOCK_WITH_RESTRICTED_VAR_USAGE, 748, 70);
-        Assert.assertEquals(result.getErrorCount(), i - 3);
-        Assert.assertEquals(result.getWarnCount(), 3);
+        Assert.assertEquals(result.getErrorCount(), i);
+        Assert.assertEquals(result.getWarnCount(), 0);
     }
 
     @Test
     public void testIsolatedObjects() {
         CompileResult compileResult = BCompileUtil.compile("test-src/isolated-objects/isolated_objects.bal");
         Assert.assertEquals(compileResult.getErrorCount(), 0);
-
-        int i = 0;
-        validateWarning(compileResult, i++, "concurrent calls will not be made to this method since the method is not" +
```

Comment:
Isn't it better to add the method name to this warning.

Reply:
Isn't the name redundant since the error is associated with the location (pos)? And resource methods don't quite have a name anyway.